src/coreclr/vm/ceeload.h

```diff
@@ -339,6 +339,11 @@ struct VASigCookie
     Volatile<PCODE> pNDirectILStub;         // will be use if target is NDirect (tag == 0)
     PTR_Module      pModule;
     Signature       signature;
+    AllocMemTracker amt;
```

Comment:
`AllocMemTracker` is meant to be used as local temp variable to free multiple allocations when the operation fails. It is not meant to be used in durable structure like this.